### PR TITLE
executor: support Chunk for ExplainExec

### PR DESCRIPTION
**Pull request by zz-jason**
to #5261

### CODE REVIEW

File: plan/planbuilder.go [hunk 1]
```diff
@@ -1058,12 +1058,11 @@ func (b *planBuilder) buildExplain(explain *ast.ExplainStmt) Plan {
 	p := &Explain{StmtPlan: pp}
 	switch strings.ToLower(explain.Format) {
 	case ast.ExplainFormatROW:
-		retFields := []string{"id", "parents", "children", "task", "operator info"}
+		retFields := []string{"id", "parents", "children", "task", "operator info", "count"}
```
Comment (shenli) [inline plan/planbuilder.go]: Why this is changed?

Comment (zz-jason) [reply plan/planbuilder.go]: I think it's ok to make the column type for `count` to be string, so we can store all the pre-generated explain information to a `[][]string`, not a `[][]Datum`

**Comment by shenli** [comment]
LGTM

**Pull request merged**